ui/app/templates/components/transit-key-action/hmac.hbs

```diff
@@ -21,11 +16,22 @@
     <KeyVersionSelect @key={{@key}} @onVersionChange={{action (mut @key_version)}} @key_version={{@key_version}} />
     <div class="field">
       <div id="input-control" class="control is-relative">
-        <JsonEditor @title="Input" @valueUpdated={{action (mut @input)}} @mode="ruby" @data-test-transit-input="input" />
+        <JsonEditor
+          @title="Input"
+          @valueUpdated={{action (mut @trackedInput)}}
```

Comment:
Should these be updated to the `fn` helper?

Reply:
As discussed in slack, will do for actions that are not using value=target.value. That's a larger overhaul of passing back the event to an action (which it's not really in this case but a prop instead).